sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -2087,12 +2089,15 @@ def __init__(
         GCP expansion service. Used for STORAGE_WRITE_API method.
       max_insert_payload_size: The maximum byte size for a BigQuery legacy
         streaming insert payload.
-      cdc_writes_with_primary_key: Configure the usage of CDC writes on BigQuery
-        and sets the primary key using the column names from the argument.
-        The destination table (or tables if using dynamic destionations) will
-        be created in case of using CREATE_IF_NEEDED mode, and StorageWrite API
-        at least once mode will be configured. Used for STORAGE_WRITE_API
-        method.
+      use_cdc_writes: Configure the usage of CDC writes on BigQuery.
+        The argument can be used by passing True and the Beam Rows will be
+        sent as they are to the BigQuery sink which expects a 'record'
+        and 'cdc_info' properties.
+        Used for STORAGE_WRITE_API, working on 'at least once' mode.
```

Comment:
"If True, your input elements are expected to have a 'record' field representing the record to write, and a 'cdc_info: {mutation_type: <str>, change_sequence_number: <str>}' field representing the mutation information."